singer_sdk/sinks/core.py

```diff
@@ -67,46 +67,36 @@ def validate(self, record: dict[str, t.Any]) -> None:
         """
 
 
-class JSONSchemaValidator(BaseJSONSchemaValidator):
+class FastJSONSchemaValidator(BaseJSONSchemaValidator):
```

Comment:
Should we keep the `JSONSchemaValidator` class? It doesn't hurt, I think.

Reply:
I don't see that it would hurt keeping `JSONSchemaValidator`.  Let me change it back.